baseband/mark4/payload.py

```diff
@@ -210,6 +211,8 @@ class Mark4Payload(VLBIPayloadBase):
                  (8, 2, 2): decode_8chan_2bit_fanout2,
                  (8, 2, 4): decode_8chan_2bit_fanout4}
 
+    _sample_shape_cls = namedtuple('sample_shape', 'nchan')
```

Comment:
Should we do `namedtuple('SampleShape', 'nchan')`

Reply:
PEP8 agrees, so renamed.